### DISCUSSION TITLE
Repeater with table layout

### DISCUSSION THREAD

**Discussion by abudawud** [filamentphp/filament#2513] [Ideas]
I'm very enjoying this laravel package where i can create CRUD form completely with easy, customable, and fast. `Repeater` is a feature in this package that amazing me by it simplicity to create one to many form quickly. 

Now the `Repeater` only have single layout, it was Repeater that wrapped with card component and every field inside this card have it own label, i think this layout take many space. I hope this awesome package also have repeater with table form (like this below example perhaps) so the repeater form part not take a lot space and more simple. So when i create Repeater, i can choose which layout will be used.

## Proposed `Repeater` layout

![image](https://user-images.githubusercontent.com/15796342/169760278-504b57c4-b5cb-4def-b4df-9fb0705081b7.png)

## Available `Repeater` layout

![image](https://user-images.githubusercontent.com/15796342/169762358-8131a900-d93e-4b02-828b-79c9356cabaa.png)

Thanks

**Comment by danharrin** [comment]
Yeah it's planned for the future I think

**Reply by abudawud** [reply]
glad to hear that @danharrin

**Comment by abudawud** [comment]
@danharrin would you like to give me an idea about this and i will attempt to open pull request for that

Thanks.

**Reply by danharrin** [reply]
https://github.com/icetalker/filament-table-repeater

**Reply by studiowizjo** [reply]
There is https://github.com/filamentphp/filament/pull/2666 with something similiar. Maybe finish this and then add to it possibility to use other than text fields?
